assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -118,9 +118,11 @@
 
 		.googlesitekit-settings-notice__learn-more {
 			flex-shrink: 0;
+			text-align: right;
```

Comment:
Only neccessary for `&--single-row` as `&--multi-row` already has this aligned right